rpcs3/util/vm_native.cpp

```diff
@@ -451,6 +451,44 @@ namespace utils
 		{
 			fprintf(stderr, "Reading /proc/sys/vm/overcommit_memory: %c", c);
 		}
+#else
+		int vm_overcommit = 0;
+		auto vm_sz = sizeof(int);
+
+#ifdef __NetBSD__
+		// Always ON
+		vm_overcommit = 0;
+#elif defined(__FreeBSD__)
+		int mib[2]{CTL_VM, VM_OVERCOMMIT};
+		if (::sysctl(mib, 2, &vm_overcommit, &vm_sz, NULL, 0) != 0)
+			vm_overcommit = -1;
+#else
+		if (::sysctlbyname("vm.overcommit", &vm_overcommit, &sz, NULL, 0) != 0)
```

Comment:
`sysctlbyname` [doesn't exist](https://man.openbsd.org/sysctlbyname) on OpenBSD or any non-BSD system (e.g., Solaris), so the build will fail even if runtime result is ignored.